api/HTMLMediaElement.json

```
@@ -496,10 +496,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "6"
+              "version_added": "4"
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "3.2"
```

```suggestion
              "version_added": "3"
```

Based on http://mdn-bcd-collector.appspot.com/tests/api/HTMLMediaElement in iOS 3 simulator.